style.css

```diff
@@ -102,3 +137,246 @@ span {
   color: #505f79;
   width: 1.2rem;
 }
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  height: auto;
+  gap: 2rem;
+  margin-left: 1rem;
+}
+
+.card1,
+.card2,
+.card3,
+.card4 {
+  display: flex;
+  flex-direction: column;
+  border: 0.063rem solid #dfe1e6;
+  border-radius: 1rem;
+  background: #fff;
+  height: auto;
+  padding: 1rem;
+  width: 84%;
+}
```

Comment:
- [ ] Please kindly ensure to design this cards in accordance to the given template most especially the paragraph, tech stacks and see projects button. 👍 

## Your Design
<img width="256" alt="Untitled" src="https://github.com/efmomoh/Portfolio-setup-and-mobile-first/assets/103386183/c47e1c7a-5cf4-40d0-9891-18e317dd46a3">

## Figma Design
<img width="353" alt="Untitled" src="https://github.com/efmomoh/Portfolio-setup-and-mobile-first/assets/103386183/7d349d7a-aa2f-4054-b9e4-c1a48f2f0e95">